scipy/special/tests/test_hypergeometric.py

```diff
@@ -15,3 +18,47 @@ def test_negative_x(self):
 
     def test_special_cases(self):
         assert sc.hyperu(0, 1, 1) == 1.0
+
+
+class TestHyp1f1:
```

Comment:
Minor: maybe still inherit from object for a while longer. Or maybe it's indeed time to make it drop.

Reply:
Added the inheritance, though it makes me sad to do so. (We are so close to the future.)